tests/http/src/test/java/org/mule/test/module/http/functional/AbstractClientTlsRestrictedTlsProtocolsAndCiphersTestCase.java

```diff
@@ -35,7 +38,13 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-public abstract class AbstractClientTlsRestrictedTlsProtocolsAndCiphersTestCase extends AbstractHttpTestCase
+/**
+ * Needs to access resources for validating different TLS scenarios and it cannot be done with
```

Comment:
Create a abstract class to avoid duplicating this javadoc and the getAnnotatedExtensionClasses in the two classes that require this changes.